src/AzureAIProxy/Services/ICatalogService.cs

```diff
@@ -8,5 +8,6 @@ public interface ICatalogService
         string eventId,
         string deploymentName
     );
+    Task<List<Deployment>> GetEventAssistantEndpoint(string eventId);
```

Comment:
In some places we have the `Async` suffix on types returning `Task<T>` and in some places we don't - let's start being more consistent.